api/proto/info/info.proto

```diff
@@ -208,6 +208,7 @@ message ZInfoDevice {
 
   SystemAdapterInfo systemAdapter = 24;
   uint32 restartCounter = 25; // Number of times zedagent has restarted i.e., device reboot
+  string HwSecurityModule = 26; //State of hardware security modules, like TPM
```

Comment:
Why not make this an enum, with values like "Unknown", "NotFound", "Disabled", and "Enabled"?
Then add a separate string with HwSecurityModuleName which is an informational string about the hardware

Reply:
@nordmark Updated the PR, please have a look.